### PR TITLE
Reference section for all screens

### PR DESCRIPTION
**Pull request by Chartine02**
Implemented the reference section for large screens

### CODE REVIEW

**Comment by beylar** [comment]
please correct the padding and alignment of the text

**Review by gloriaumutoni — APPROVED** [review]
looks good

**Review by josueahadi — CHANGES_REQUESTED** [review]
@aimee-annabelle @Chartine02 The reference section looks superb on large screens. You can carry on and work on responsiveness for tablet and mobile screens.

**Review by beylar — APPROVED** [review]

**Review by Ntwalii — APPROVED** [review]
Well implemented.

**Review by josueahadi — APPROVED** [review]
It works as expected...kudos! :)

File: src/sections/reference.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<div class="w-full relative flex">
+  <img
+    src="../src/images/yellow-bg.png"
+    alt=""
```
Comment (Kingcedru) [inline src/sections/reference.html]: Add image description

File: src/sections/reference.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    <a
+      href="https://budweiserbudvar.com/?redirect_from=de&redirect_code=3"
+      target="_blank"
+    >
+      Budweiser Budvar</a
+    >
+    <a href="https://www.instagram.com/chungkingnoodles/?hl=de" target="_blank"
+      >ChungKing Noodles</a
+    >
+    <a href="https://designpreis-brandenburg.de/    " target="_blank"
```
Comment (Kingcedru) [inline src/sections/reference.html]: Remove space at the end of the href link.

**Review by Kingcedru — CHANGES_REQUESTED** [review]
Changes needed

**Review by Kingcedru — APPROVED** [review]
Done

**Pull request merged**